third_party/musl/arch/x86_64/bits/alltypes.h

```diff
@@ -7,7 +7,11 @@
 
 #ifndef __cplusplus
 #if defined(__NEED_wchar_t) && !defined(__DEFINED_wchar_t)
+#if defined(STARBOARD)
```

Comment:
I recommend not doing this include here. Simply set `-__DEFINED_wchar_t` from GN ( and  optionally define it in `features.h` or maybe in starboard headers if required )